pkg/oc/cli/set/volume.go

```diff
@@ -639,6 +639,11 @@ func (a *AddVolumeOptions) createClaim() *kapi.PersistentVolumeClaim {
 		},
 	}
 	if len(a.ClaimClass) > 0 {
+		// NULL is a special value. It sets storageClassName: "". This is useful when cluster has "default" storageclass but you want to use static PV.
+		// For further detail, refer to https://kubernetes.io/docs/concepts/storage/persistent-volumes/#class-1
```

Comment:
don't see `NULL` in that linked page. should I?

Reply:
Where does the "NULL" come from?

Reply:
`NULL` here is not acceptable, there is a way to check whether a flag was provided by the user, and we should just accept empty string for `--claim-class`, use `cmd.Flags().Lookup("claim-class").Changed` for that.

Reply:
@stts @soltysh Thank you. Updated. I used `NULL`, because I did not
know `cmd.Flags().Lookup("claim-class").Changed` and thought that need
to use some "special" value.